modules/react-components/src/components/file-picker/file-picker.tsx

```diff
@@ -950,5 +956,143 @@ export class CertFileStrategy implements PickerStrategy<CertificateDecodeResult>
             });
         });
     }
+}
+
+export interface CSVResult {
+    headers: string[];
+    items: string[][];
+}
+
+export class CSVFileStrategy implements PickerStrategy<CSVResult> {
```

Comment:
Shall we move this class to a separate file? Is this how we have defined models and classes for other react components as well?

Reply:
All the other file strategies are also defined in the same file.